app/route.py

```diff
@@ -0,0 +1,17 @@
+from flask import Blueprint, jsonify        
+from .schema import User
+from . import mongo
+
+main_blueprint = Blueprint('main', __name__)
+
```

Comment:
make below functions in views.py, create here blueprint only

Reply:
Okay sir